Firestore/core/src/firebase/firestore/local/query_data.h

```diff
@@ -92,14 +93,14 @@ class QueryData {
    */
   static QueryData Invalid();
 
-  /** The query being listened to. */
-  const core::Query& query() const {
-    return query_;
+  /** The target being listened to. */
+  const std::shared_ptr<const core::Target> target() const {
```

Comment:
As noted elsewhere, this would be better off as a `const core::Target&`, hiding the `std::shared_ptr` detail (if we even need that).